src/sentry/search/events/base.py

```diff
@@ -120,6 +122,10 @@ def _resolve_user_display_alias(self, _: str) -> SelectType:
         columns = ["user.email", "user.username", "user.ip"]
         return Function("coalesce", [self.column(column) for column in columns], USER_DISPLAY_ALIAS)
 
+    def _resolve_transaction_status(self, _: str) -> SelectType:
+        column = self.column(TRANSACTION_STATUS_ALIAS)
+        return Function("coalesce", [column], TRANSACTION_STATUS_ALIAS)
```

Comment:
@getsentry/visibility is there a better way to do this? I just want to display `transaction_status as transaction.status`, but I couldn't identify a way. Currently resorting to using `coalesce` to do `as` for me.

Reply:
d'oh, reviewed without seeing this sorry, I think our existing `toUInt8` hack that we do on issues is fine until we get column aliasing

Reply:
Good call! Switched to `toUInt8`, we'll also be able to replace this with column aliasing.